packages/lit-html/package.json

```diff
@@ -127,20 +127,17 @@
     }
   },
   "scripts": {
-    "build": "npm run clean && npm run build:ts && npm run build:version-stability-test && rollup -c",
-    "build:watch": "rollup -c --watch",
-    "build:ts": "tsc --build && treemirror development . '**/*.d.ts{,.map}'",
-    "build:ts:watch": "tsc --build --watch",
-    "build:version-stability-test": "rollup -c rollup-version-stability-test.config.js",
-    "check-version": "node scripts/check-version-tracker.js",
-    "checksize": "rollup -c --environment=CHECKSIZE",
-    "clean": "rm -rf {async-directive,directive-helpers,directive,experimental-hydrate,lit-html,polyfill-support,private-ssr-support,static}.{js,js.map,d.ts} test/ directives/ development/ version-stability-test/ *.tsbuildinfo",
-    "dev": "scripts/dev.sh",
```

Comment:
Would this now be equivalent to `npm test watch`?

Reply:
Yep!